bolt-cli/src/cli.rs

```diff
@@ -228,6 +228,16 @@ pub enum EigenLayerSubcommand {
         expiry: U256,
     },
 
+    /// Deregister an EigenLayer operator from the bolt AVS.
```

Comment:
other commands are marked with "Step N", can you either add it here or remove it everywhere else to keep consistency in the doc comments?